tests/drivers/adc/adc_api/boards/cc1352r1_launchxl.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 Florin Stancu <niflostancu@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/adc/adc.h>
+
+/ {
+	zephyr,user {
+		/* 0x5 is ADC_COMPB_IN_VDDS (power supply measurement) */
+		io-channels = <&adc0 0x5>;
```

Comment:
The ADC API test does not use channel definitions from devicetree, so adding these overlay files is misleading.

Reply:
I didn't enable the `adc` node inside the boards `dts`, now I've done that and eliminated the overlays from the test. Hope it's okay now.